src/screens/LoginScreen.js

```diff
@@ -108,15 +101,15 @@ export default class LoginScreen extends React.Component {
    * Routes the user to the app.
    */
   navigateToApp(navProps) {
-    this.props.navigation.navigate('App', navProps);
+    this.props.navigation.navigate("Home", navProps);
```

Comment:
this was a bug in the old version--the navprops don't get passed to the screen if you use `App`, use `Home` instead

Reply:
Lol I have no idea how it loaded :'(